AppCenterDataStorage/AppCenterDataStorage/Internal/Model/MSPendingOperation.h

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#import <Foundation/Foundation.h>
+
+@interface MSPendingOperation : NSObject
```

Comment:
Why not use MSDocumentWrapper instead? I don't see extra properties in this object.

Reply:
we can , but this is more clear and there are more than needed offered in the Wrapper